torch/quantization/_numeric_suite.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+
+def find_match(str_list, key_str, postfix):
+    split_str = key_str.split(".")
+    if split_str[-1] == postfix:
+        match_string = "".join(key_str.split(".")[0:-1])
+        for s2 in str_list:
+            pattern1 = "".join(s2.split(".")[0:-1])
+            pattern2 = "".join(s2.split(".")[0:-2])
+            if match_string == pattern1:
+                return s2
+            if match_string == pattern2:
+                return s2
+    else:
+        return None
+
+
+def compare_weights(float_dict, quantized_dict):
```

Comment:
This is a user facing API so comments in standard pytorch format are needed so that doc strings work correctly.  For example, look at: https://github.com/pytorch/pytorch/blob/373dc7c8efc76b604e8148d51b78191404a2f353/torch/quantization/quantize.py#L125